src/main/java/core/basesyntax/strategy/BalanceOperationCodeParser.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperationCodeParser implements OperationCodeParser {
```

Comment:
actually i dont know the reason why you've renamed this classes. it's really ["Strategy" design pattern](https://refactoring.guru/design-patterns/strategy), no need to rename. all fine
```suggestion
public class BalanceOperationStrategy implements OperationStrategy {
```